apps/typegpu-docs/src/components/ExampleLayout.tsx

```diff
@@ -66,10 +66,10 @@ function SideMenu() {
         menuShownMobile
           ? 'absolute inset-0 z-50 w-full md:static'
           : 'hidden md:flex',
-        'box-border flex flex-col bg-white md:rounded-2xl md:w-[18.75rem] p-5 gap-5 overflow-auto',
+        'box-border flex flex-col bg-white md:rounded-2xl md:w-[18.75rem] pl-5 py-5 gap-5 overflow-y-auto',
```

Comment:
```suggestion
        'box-border flex flex-col bg-white md:rounded-2xl md:w-[18.75rem] pl-3 py-5 gap-5 overflow-y-auto',
```
makes it symmetrical, though maybe there's a better way. 
and then also mr-3 for header etc. 😵‍💫

Reply:
The scrollbar ruins everything